src/kv-installation-store.ts

```diff
@@ -156,12 +161,20 @@ export class KVInstallationStore<E extends SlackOAuthEnv> implements Installatio
           }
         }
 
-        const botClient = new SlackAPIClient(bot?.bot_token, {
+        if (!bot?.bot_token) {
+          throw new Error("Failed to find bot_token in bot installation");
```

Comment:
I'd like to adjust the error message here, but will make the change after merging this